css/style.css

```diff
@@ -59,18 +67,140 @@ body {
 
 /* Center the container */
 #container {
-  width: 960px;
   margin: auto;
+  padding-right: 20px;
+  padding-left: 40px;
+
 }
 
-header {
+div{
+
+  background-color: #033048;
+  
+  }
+.leftfloat{
+   width:700px;
+   color:white;
+   font-size: 40px;
+   font-weight: bold;
 
 }
+.leftfloat2{
```

Comment:
Rather than calling these classes: `leftfloat` and `leftfloat2`, I recommend coming up with more descriptive names like `primary-content` and `sidebar`